backend/src/v5/routes/teamspaces/projects/models/drawings/revisions.js

```diff
@@ -0,0 +1,326 @@
+/**
```

Comment:
how different is this file vs the container one?

Should we be moving this to common (and just don't cater for federations?)

Reply:
I've done this but I've had to change the downloadOriginalFiles drawing endpoint from 

/teamspaces/{teamspace}/projects/{project}/drawings/{drawing}/revisions/{revision}/files/original
to
/teamspaces/{teamspace}/projects/{project}/drawings/{drawing}/revisions/{revision}/files

to match the containers one... are we ok with this ?

Reply:
Can we keep them different?

Reason being, the svgs are going to get pulled from `/teamspaces/{teamspace}/projects/{project}/drawings/{drawing}/revisions/{revision}/files/image`

So it's going to be very ambiguous